### ISSUE TITLE
Training Loss Problem

### ISSUE THREAD

**Issue by albertchristianto** [pat-coady/tiny_imagenet#1]
Hello @pat-coady ,
I have tried to run your code and also write my own code about tiny ImageNet dataset.
But the loss can't be converged (either your code and mine). Could you share some tips in training a network with tiny ImageNet?
Thank you very Much

**Comment by com9009** [comment]
I have the same problem like this.
Loss doesn't converge and learning rate become 0.
@pat-coady Could you share the option or setting guide to get accuracy as you post early?
